src/main/java/com/bymarcin/openglasses/network/packet/GlassesEventPacket.java

```diff
@@ -16,17 +13,33 @@ public class GlassesEventPacket extends Packet<GlassesEventPacket, IMessage> {
 
     public static enum EventType {
         EQUIPED_GLASSES,
-        UNEQUIPED_GLASSES;
+        UNEQUIPED_GLASSES,
+        INTERACT_OVERLAY,
+        KEYBOARD_INTERACT_OVERLAY,
```

Comment:
These should just be separate packets with constructors that don't take the missing arguments